platform_it/src/test/java/com/splicemachine/test/SpliceTestPlatformConfig.java

```diff
@@ -294,7 +294,9 @@ public static Configuration create(String hbaseRootDirUri,
         config.set("splice.authentication.impersonation.users", "dgf=splice;splice=*");
         config.setBoolean("splice.authentication.impersonation.enabled", true);
         config.set("splice.authentication.ldap.mapGroupAttr", "jy=splice,dgf=splice");
-   //     config.setBoolean("splice.metadataRestrictionEnabled", true);
+        // config.set("splice.authorization.scheme", "RANGER");
+        // config.set("splice.metadataRestrictionEnabled", "RANGER");
+
 
```

Comment:
Minor comments: remove unused code

Reply:
@jyuanca The reason to keep the commented out lines is to make it easier for future ranger testing on standalone system. I'll put a comment there indicating that there are the two parameters to set when testing ranger authorization on standalone system.